cloud-native-runtimes/install-cnrt.hbs.md

```diff
@@ -65,41 +65,37 @@ To install Cloud Native Runtimes:
 
         ```console
         ---
-        # if deploying on a local cluster such as Kind. Otherwise, you can remove this field
-        provider: local
+        domain_name: example.com
+        ingress:
+        external:
+            namespace: tanzu-system-ingress
+        internal:
+            namespace: tanzu-system-ingress
         ```
 
         >**Note:** For most installations, you can leave the `cnr-values.yaml` empty, and use the default values.
 
         If you are running on a single-node cluster, such as kind or minikube, set the `provider: local`
-        option. This option reduces resource requirements by using a HostPort service instead of a
-        LoadBalancer and reduces the number of replicas.
+        option. This option reduces resources requests for CNR deployments.
+        If you are running on a multinode cluster, do not set `provider`.
 
         Cloud Native Runtimes reuses the existing `tanzu-system-ingress` Contour installation for
-        external and internal access when installed in the `light` or `full` profile.
+        external and internal access.
         If you want to use a separate Contour installation for system-internal traffic, set
```

Comment:
I don't think this is possible with TAP Contour. Internal and External namespaces must be the same: https://github.com/pivotal/docs-tap/blob/main/cert-mgr-contour-fcd/install-cert-mgr.hbs.md#install-contour

Reply:
really? I thought that if somehow someone had another contour controller on the cluster with tap contour, there is nothing stopping them from reconfiguring CNR to use one for internal and one for external

Reply:
ah, wait, I see how that line is misleading. I will remove it.